config/main.py

```diff
@@ -662,6 +665,96 @@ def del_vlan_member(ctx, vid, interface_name):
     db.set_entry('VLAN_MEMBER', (vlan_name, interface_name), None)
 
 
+def vrf_add_management_vrf():
+	"""Enable management vrf"""
+
+	cmd = 'sonic-cfggen -d --var-json "MGMT_VRF_CONFIG"'
```

Comment:
Is it possible to read this address from the config DB?
Why do you read using sonic-cfggen, but write directly to DB?